mappings/net/minecraft/advancement/criterion/VillagerTradeCriterion.mapping

```diff
@@ -2,7 +2,16 @@ CLASS net/minecraft/class_2140 net/minecraft/advancement/criterion/VillagerTrade
 	CLASS class_2142 Conditions
 		FIELD field_9766 villager Lnet/minecraft/class_2073;
 		FIELD field_9767 item Lnet/minecraft/class_2048;
```

Comment:
Just noticed `villager` and `item` are swapped here. Would be cool if you could change that 😃